test/e2e/mapml-viewer/mapml-viewerHeightAndWidthAttributes.test.js

```diff
@@ -0,0 +1,67 @@
+import { test, expect, chromium } from '@playwright/test';
+
+test.describe("Adding Width and Height Attribute to mapml-viewer", () => {
+  let page;
+  let context;
+  test.beforeAll(async function() {
+    context = await chromium.launchPersistentContext('');
+    page = context.pages().find((page) => page.url() === 'about:blank') || await context.newPage();
+    await page.goto("mapml-viewerHeightAndWidthAttributes.html");
+  });
+  test.afterAll(async function () {
+    await context.close();
+  });
+
+  test("Setting Width and Height Attibute to mapml-viewer", async () => {
```

Comment:
I think this test is not setting width and height attributes, maybe you can rename this and add another test that can test if setting width and height attribute to mapml-viewer works - same thing for web-map.js